PRINCIPLES.md

```diff
@@ -27,31 +27,37 @@ These principles were derived from modern software operations but are rooted in
 
     The only mechanism through which the system is intentionally operated on is through these principles.
 
-## Notes
-
-### Principle 3 Notes
-
-- These differences could be due to the actual state drifting from the desired state, or the desired state changing intentionally.
-- The source of drift doesn't matter. Contrary to CIops, _any_ drift will trigger a reconciliation
-
-### Principle 4 Notes
+## Glossary
 
-- We talk here about "regular operations." In an emergency, other modes of operations, e.g. manual intervention, should be considered - followed by a reconciliation of the "tainted" system with the declared state. → resolve the conflict between "GitOps principle" and "I need to deal with problems that GitOps doesn't cover"
+- ### Break Glass
 
-## Glossary
+    The temporary suspension of GitOps principles, most often pausing automated _Reconciliation_.
+    While these principles apply to "regular operations," it may sometimes be necessary to pause or sidestep them during an emergency such as incident management.
+    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the manually changed system with the updated declarations again.
+    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
 
 - ### Continuous
 
-    By "continuous" we adopt the industry standard term to mean reconciliation continues to happen, not that it must be instantaneous.
+    By "continuous" we adopt the industry standard term to mean _Reconciliation_ continues to happen, not that it must be instantaneous.
 
 - ### Declarative Description
 
-    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating between configuration - the desired state - and implementation - the commands, API calls, scripts ... that actually achieve the desired state described in the declarative description.
+    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating between configuration (the desired state) and implementation (the commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
```

Comment:
nits, drop "between", and use "from" rather than "and" (i.e. separating configuration from the implementation)

Reply:
fixed!